tests/dummy/app/templates/docs/testing-intro.md

```diff
@@ -6,11 +6,45 @@ Just like other addons, engines must be tested differently depending upon whethe
 
 * To test a standalone engine, use the dummy app, just like any other standalone addon project.
 
-Ember Engines comes with a **set of test helpers that can be used in Unit/Integration tests**:
+## Setup
+
+### Lazy Engines
+
+If you have a lazy engine, you'll need to ensure that your `tests/test-helper.js` is configured to preload your engine's assets:
+
+```js
+// <app-name>/tests/test-helper.js
+import Application from '../app';
+import config from '../config/environment';
+import { setApplication } from '@ember/test-helpers';
+import { start } from 'ember-qunit';
+import preloadAssets from 'ember-asset-loader/test-support/preload-assets';
+import manifest from '<app-name>/config/asset-manifest';
+
+setApplication(Application.create(config.APP));
+
+preloadAssets(manifest).then(start); // This ensures all engine resources are loaded before the tests
+```
+
+### Rendering/Unit tests
```

Comment:
It matchs the test types on the official guides - https://guides.emberjs.com/release/testing/test-types/

let me know if I can do something else to make it better @jenweber